subresource-loading-server.md

```diff
@@ -1,7 +1,47 @@
 # Bundle preloading for servers
 
-<!-- TODO focus this document on resource bundling from the point of view of the server: response behaviour, practical considerations, etc. -->
+This section discusses the proposal from the point of view of primary and intermediate servers.
 
-Considerations about bundle preloading from the point of view of the server. **TODO*
+## Basic operation
+
+As explained in the [overview](./overview.md), the server receives a request for a bundled response that targets the Web bundle file and includes a `Bundle-Preload` header listing the requested resources.
+
+The response from the server will be in the [bundled response](https://github.com/wpack-wg/bundled-responses) format, containing HTTP responses for each of the requested URLs.
+
+The server may return more resources than those originally requested. The browser may choose to cache those resources; it it does so, it should not make those additional resources available to the page until they are themselves included in a `bundlepreload` declaration.
+
+### Response headers
+
+The server response will include the header `Vary: Bundle-Preload`, to signify that the bundle should be requested again for a different value of the `Bundle-Preload` resource list.
+
+`Cache-Control: private` may be used to indicate that the response may be stored only by a browser's cache. This directive is intended as a hint to intermediate servers that they should not try to store many variants of the same bundle.
+
+`Cache-Control: private bundled` (note the new value) may be used to let bundling-aware intermediate servers understand that the data it is not private and may be cached, but probably with a bundling-specific strategy.
```

Comment:
I don't know as much as I should about header parsing, but I believe `private bundled` has different semantics than `private,bundled` and we would want `private,bundled` I believe for backwards compatibility. `bundled` without `private` would also be a good case to discuss; I think there are a few options for what an intermediary could do there. The simplest would be to consider it a no-op or an error.

Reply:
Yes, eventually we will need to specify the semantics and interactions of these values with more detail.

I will update this line to `Cache-Control: private, bundled`